Inflector/DoctrineInflector.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the FOSRestBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\RestBundle\Inflector;
+
+use Doctrine\Common\Inflector\Inflector;
+
+/**
+ * Inflector object using the Doctrine/Inflector.
+ *
+ * @author Mark Kazemier <Markkaz>
+ */
+class DoctrineInflector implements InflectorInterface
+{
+    public function __construct()
```

Comment:
Not yet I forgot to extend the old class which triggers a deprecation in its constructor.